docs/readmes/contributing/contribute_conventions.md

```diff
@@ -0,0 +1,188 @@
+- Omit trailing slash of directory paths, except where semantically meaningful
+- Don't terminate service names with `d`
+- Use [Americanized spellings](https://en.wikipedia.org/wiki/Wikipedia:List_of_spelling_variants)
+    - marshaling, marshaled
+    - canceling, canceled, cancellation
+
+## Documentation
+
+### All documentation
+
+The end-goal is uniform, approachable documentation, especially in the Docusaurus
+
+- Write in [plain English](https://www.plainenglish.co.uk/how-to-write-in-plain-english.html)
+    - Short sentences
+    - Active verbs
+    - Use "you" and "we"
+    - Avoid nominalisations
+    - Use lists
+- [Use descriptive hyperlink text](https://www.nngroup.com/articles/writing-links/)
+    - Don't use "here" as the text for a hyperlink
+- Consistent capitalization and spelling
+    - Magma-specific
+        - Orc8r, Orchestrator
+        - NMS
+        - Magma
+        - Mconfig
+    - Magma-adjacent
+        - K8s, Kubernetes
+        - Helm
+        - eNodeB
+    - Everyday words
+        - metadata
+        - use-case
+    - Magma service names
+        - state, subscriberdb, lte, etc.
+- Use long form of CLI flags
+    - `--deployment` rather than `-d`.
+
+### Markdown
+
+Consider the following Markdown conventions
+
+- Don't wrap long lines
+- Don't preface lists with punctuation
+    - Good: `Consider the following Markdown conventions`
+    - Bad: `Consider the following Markdown conventions:`
+    - Bad: `Consider the following Markdown conventions,`
+    - Bad: `Consider the following Markdown conventions.`
+- Default to sentence-casing listables
+    - Good: `- Default to sentence-casing listables`
+    - Bad: `- default to sentence-casing listables`
+- Use hyphens for unordered lists, not asterisks
+    - Good: `- Magma`
+    - Bad: `* Magma`
+- Use flat apostrophes and quotes, not curly
+    - Good: `Magma's`, `"Magma"`
+    - Bad: `Magma’s`, `“Magma”`
+- Title-case H1 headers, sentence-case H2 and lower headers
+    - Good: `# Code Conventions in Magma`, `## Code conventions in Magma`
+    - Bad: `# Code conventions in Magma`, `## Code Conventions in Magma`
+- Use space-padded double-hyphen to approximate an [em dash](https://www.thepunctuationguide.com/em-dash.html)
+    - Good: `Magma -- an open source project -- has code conventions`
+    - Bad: `Magma--an open source project--has code conventions`
+    - Bad: `Magma - an open source project - has code conventions`
+
+## Languages
+
+### Go
+
+Orc8r's cloud code has some basic [CI lint checks](https://github.com/magma/magma/blob/master/.golangci.yml). The [Go style guide](https://github.com/golang/go/wiki/CodeReviewComments) and anything concrete from [Effective Go](https://golang.org/doc/effective_go) are authoritative. Aside from those, consider the following conventions
+
+**General**
+- Familiarize yourself with these [3 common Go landmines](https://gist.github.com/lavalamp/4bd23295a9f32706a48f)
```

Comment:
Would be good to explicitly state the expectation around import order as I see this violated on first iterations of PRs often.

Reply:
This is something I'm going to switch to automatically+explicitly enforce relatively soon, so intentionally omitted for now